Source/DafnyCore/Resolver/ModuleResolver.cs

```diff
@@ -1869,47 +1937,55 @@ public partial class ModuleResolver {
     private void FigureOutNativeType(NewtypeDecl dd) {
```

Comment:
I would think FigureOutNativeType would be a PreCodeGeneration step, not part of resolution.